docs/src/apimanual.md

```diff
@@ -1012,38 +1012,53 @@ By convention, optimizers should not be exported and should be named
 
 ### Testing guideline
 
-The skeleton below can be used for the wrapper test file of a solver name `FooBar`. A few bridges are used to give examples, you can find more bridges in the [Bridges](@ref) section.
+The skeleton below can be used for the wrapper test file of a solver named `FooBar`.
 ```julia
 using MathOptInterface
 const MOI = MathOptInterface
 const MOIT = MOI.Test
+const MOIU = MOI.Utilities
 const MOIB = MOI.Bridges
 
-const optimizer = FooBarOptimizer()
+import FooBar
+const optimizer = FooBar.Optimizer()
+MOI.set(optimizer, MOI.Silent(), true)
+
+@testset "SolverName" begin
+    @test MOI.get(optimizer, MOI.SolverName()) == "FooBar"
+end
+
+@testset "supports_default_copy_to" begin
+    @test MOIU.supports_default_copy_to(optimizer, false)
+    # Use `@test !...` if names are not supported.
+    @test MOIU.supports_default_copy_to(optimizer, true)
+end
+
+const bridged = MOIB.full_bridge_optimizer(optimizer, Float64)
 const config = MOIT.TestConfig(atol=1e-6, rtol=1e-6)
 
-@testset "MOI Continuous Linear" begin
-    # If `optimizer` does not support the `Interval` set,
-    # the `SplitInterval` bridge can be used to split each `f`-in-`Interval(lb, ub)` constraint into
-    # a constraint `f`-in-`GreaterThan(lb)` and a constraint `f`-in-`LessThan(ub)`
-    MOIT.contlineartest(MOIB.SplitInterval{Float64}(optimizer), config)
+@testset "Unit" begin
+    MOIT.unittest(bridged, config)
+end
+
+@testset "Continuous Linear" begin
+    MOIT.contlineartest(bridged, config)
 end
 
-@testset "MOI Continuous Conic" begin
-    # If the solver supports rotated second order cone, the `GeoMean` bridge can be used to make it support geometric mean cone constraints.
-    # If it additionally support positive semidefinite cone constraints, the `RootDet` bridge can be used to make it support root-det cone constraints.
-    MOIT.contlineartest(MOIB.RootDet{Float64}(MOIB.GeoMean{Float64}(optimizer)), config)
+@testset "Continuous Conic" begin
+    MOIT.contlineartest(bridged, config)
 end
 
-@testset "MOI Integer Conic" begin
-    MOIT.intconictest(optimizer, config)
+@testset "Integer Conic" begin
+    MOIT.intconictest(bridged, config)
 end
 ```
 
-If the wrapper does not support building the model incrementally (i.e. with `add_variable` and `add_constraint`), the line `const optimizer = FooBarOptimizer()` can be replaced with
+If the wrapper does not support building the model incrementally (i.e. with `add_variable` and `add_constraint`), then `supports_default_copy_to` can be replaced by `supports_allocate_load` if appropriate and the line `const bridged = ...` can be replaced with
```

Comment:
Explain why this code is needed, or point to a reference on the use of bridges with solvers that don't support `default_copy_to`.